shared-ui/style/colors.ts

```diff
@@ -27,6 +27,7 @@ const colors = {
   SOORANGE: '#F2A32B',
   SPONSIBGREEN: '#27AE60',
   PURPLE: '#A0B2EF',
+  RACE: '#ffe799',
```

Comment:
Can we make this name more descriptive of the color it represents? So that when another developer looks at it they have some idea what kind of color it is heh